pandas/_libs/interval.pyx

```diff
@@ -82,12 +110,21 @@ cdef class Interval(IntervalMixin):
     Parameters
     ----------
     left : value
-        Left bound for the interval
+        Left bound for the interval.
     right : value
-        Right bound for the interval
-    closed : {'left', 'right', 'both', 'neither'}, default 'right'
+        Right bound for the interval.
+    closed : {'right', 'left', 'both', 'neither'}, default 'right'
         Whether the interval is closed on the left-side, right-side, both or
-        neither
+        neither.
+        A closed interval (in mathematics denoted by square brackets)
+        contains the edge points, i.e. the closed interval `[1, 5]` consists
+        of the points `1, 2, 3, 4, 5`.
+        An open interval (in mathematics denoted by parentheses) does not
+        contain the edge points, i.e. the open interval `(1, 5)` consists of
+        the points 2, 3, 4.
```

Comment:
I understand what's being said here, but I think this could potentially be misinterpreted as the interval being discrete instead of continuous, i.e. only containing the integer points mentioned here, whereas floats such as 1.01 and 4.99 are also included.  Might be beneficial to mention that "open" corresponds to a strict inequality (`<`) at the endpoint and "closed" corresponds to a non-strict inequality (`<=`).